### PR TITLE
Revert crate list to current status

### PR DESCRIPTION
**Pull request by sophie-h**
- *...* -> **...**
    - Larger default size for logo
    - Redirect /docs-src

#207

### CODE REVIEW

File: faq.md [hunk 1]
```diff
@@ -8,15 +8,15 @@ Here are some questions we answered a few times. If one is missing, don't hesita
 
 ## Why are releases so long?
 
-The *gtk-rs* organization is not just some *GNOME* libraries bindings in Rust but also an ecosystem. A change in a crate can force all other crates to be regenerated!
+The **gtk-rs** organization is not just some *GNOME* libraries bindings in Rust but also an ecosystem. A change in a crate can force all other crates to be regenerated!
```
Comment (GuillaumeGomez) [inline faq.md:11]: Please do it for GNOME too. :)

File: index.md [hunk 2]
```diff
@@ -30,9 +30,105 @@ The *gtk-rs* project provides safe bindings to the [Rust] language for fundament
 
 ## Available crates
 
-The following table contains the most popular crates of *gtk-rs*. More information on all existing crates is available under the corresponding *Project* links.
+The following table contains the most popular crates of **gtk-rs**. More information on all existing crates is available under the corresponding *Project* links.
```
Comment (GuillaumeGomez) [inline index.md:33]: What about creating a link to the projects anchor (if any)?

Comment (sophie-h) [reply index.md:33]: Not sure what you mean. Please not in this PR :)

File: faq.md (hunk 1 above)
Comment (sophie-h) [reply faq.md]: Done

**Pull request merged**